telegram_phone_number_checker/main.py

```diff
@@ -80,6 +90,32 @@ async def get_names(client: TelegramClient, phone_number: str) -> dict:
                     "phone": user.phone,
                 }
             )
+        if download_profile_photos is True:
```

Comment:
This block should be indented by one more level so it lives inside the `elif number_of_matches == 1:` block and doesn't interfere with the final `else`

Reply:
Fixed here f049059c2f0d8b79a844fb6f9253e0169bd66f34